src/Core__Promise.resi

```diff
@@ -194,44 +251,46 @@ all(promises)
 external all: array<t<'a>> => t<array<'a>> = "all"
 
 /**
-[all2((p1, p2))]. Like `all()`, but with a fixed size tuple of 2
+`all2((p1, p2))`. Like `all()`, but with a fixed size tuple of 2
 */
 @scope("Promise")
 @val
 external all2: ((t<'a>, t<'b>)) => t<('a, 'b)> = "all"
 
 /**
-[all3((p1, p2, p3))]. Like `all()`, but with a fixed size tuple of 3
+`all3((p1, p2, p3))`. Like `all()`, but with a fixed size tuple of 3
 */
 @scope("Promise")
 @val
 external all3: ((t<'a>, t<'b>, t<'c>)) => t<('a, 'b, 'c)> = "all"
 
 /**
-[all4((p1, p2, p3, p4))]. Like `all()`, but with a fixed size tuple of 4
+`all4((p1, p2, p3, p4))`. Like `all()`, but with a fixed size tuple of 4
 */
 @scope("Promise")
 @val
 external all4: ((t<'a>, t<'b>, t<'c>, t<'d>)) => t<('a, 'b, 'c, 'd)> = "all"
 
 /**
-[all5((p1, p2, p3, p4, p5))]. Like `all()`, but with a fixed size tuple of 5
+`all5((p1, p2, p3, p4, p5))`. Like `all()`, but with a fixed size tuple of 5
 */
 @scope("Promise")
 @val
 external all5: ((t<'a>, t<'b>, t<'c>, t<'d>, t<'e>)) => t<('a, 'b, 'c, 'd, 'e)> = "all"
 
 /**
-[all6((p1, p2, p4, p5, p6))]. Like `all()`, but with a fixed size tuple of 6
+`all6((p1, p2, p4, p5, p6))`. Like `all()`, but with a fixed size tuple of 6
 ")*/
 @scope("Promise")
 @val
 external all6: ((t<'a>, t<'b>, t<'c>, t<'d>, t<'e>, t<'f>)) => t<('a, 'b, 'c, 'd, 'e, 'f)> = "all"
 
 /**
-`done` is a safe way to ignore a promise. If a value is anything else than a promise, it will raise a type error.
+`done(p)` is a safe way to ignore a promise. If a value is anything else than a
+promise, it will raise a type error.
 */
 external done: promise<'a> => unit = "%ignore"
 
+// TODO: add docstring
```

Comment:
These are reserved for the compiler. Can be removed from here as not part of the public api.
Assuming removing them does not stop async code from compiling after opening Core: one just needs to check this. I think it's OK as the Js.* namespace is still visible.

Reply:
We can try again, but IIRC the reason I added them was because async/await didn't work otherwise. Maybe that's something that can be fixed though.

Reply:
Can you check again? And if so add an issue to the compiler.

Reply:
Yes, I'll do that

Reply:
It seems to work just fine without them. Interesting, I wonder what errored back when I decided to add them... Anyway, we can go ahead and just remove them from `Core`. I'll do that directly in main.